.circleci/config.yml

```diff
@@ -1,17 +1,75 @@
-version: 2
+
+version: 2.1
+
+orbs:
+  architect: giantswarm/architect@dev:persist-attach
```

Comment:
This will be updated once https://github.com/giantswarm/architect-orb/pull/82 is released.

Reply:
Is there anything blocking us from releasing the orb now? It should be merged now, right?

Reply:
I think so! I was just making sure I didn't make any mistakes here first. I'll tag the orb commit and get all of this merged.

Reply:
Done.